src/subdomains/core/buy-crypto/routes/crypto-route/crypto-route.controller.ts

```diff
@@ -65,23 +65,22 @@ export class CryptoRouteController {
   @Put('/quote')
   @ApiOkResponse({ type: CryptoQuoteDto })
   async getCryptoQuote(@Body() dto: GetCryptoQuoteDto): Promise<CryptoQuoteDto> {
-    dto.targetAsset ??= dto.asset;
-
```

Comment:
This line needs to be kept (or you can remove the asset field on the DTO, this route is not used anyway).